### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by mauromorales**

### CODE REVIEW

File: lib/pennyworth/constants.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2013-2014 SUSE LLC
```
Comment (tgoettlicher) [inline lib/pennyworth/constants.rb]: It's 2015 now.

File: tools/release.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    # method we can check for new bullet points since the last release.
+    fail if content.scan(/# Pennyworth .*$\n+## Version /).empty?
+    header = "\n\n\n## Version #{@release_version} - #{@release_time} - #{@mail}\n\n"
+    content = content.sub(/\n+/, header)
+    File.write(file, content)
+  end
+
+  def commit
+    Cheetah.run "git", "commit", "-a", "-m", "package #{@release_version}"
+    Cheetah.run "git", "tag", "-a", @tag, "-m", "Tag version #{@release_version}"
```
Comment (tgoettlicher) [inline tools/release.rb]: Is the uppercase `T` in `Tag version` on purpose?

Comment (mauromorales) [inline tools/release.rb]: @tgoettlicher I'd say so since it's the way we are running it in Machinery at the moment

**Comment by tgoettlicher** [comment]
lgtm

**Pull request merged**